cssparse/shared/src/test/scala/cssparse/PrettyPrinterTests.scala

```diff
@@ -0,0 +1,332 @@
+            |  display : none ;
+            |  height : 0 ;
+            |}
+            |
+            |[hidden], template{
+            |  display : none ;
+            |}
+            |
+            |a{
+            |  background-color : transparent ;
+            |}
+            |
+            |a : active , a : hover{
+            |  outline : 0 ;
+            |}
+            |
+            |abbr [title]{
+            |  border-bottom : 1px dotted ;
+            |}
+            |
+            |b , strong{
+            |  font-weight : bold ;
+            |}
+            |
+            |dfn{
+            |  font-style : italic ;
+            |}
+            |
+            |h1{
+            |  margin : .67em 0 ;
+            |  font-size : 2em ;
+            |}
+            |
+            |mark{
+            |  color : #000 ;
+            |  background : #ff0 ;
+            |}
+            |
+            |small{
+            |  font-size : 80% ;
+            |}
+            |
+            |input [type = 'checkbox'], input [type = 'radio']{
+            |  -webkit-box-sizing : border-box ;
+            |  -moz-box-sizing : border-box ;
+            |  box-sizing : border-box ;
+            |  padding : 0 ;
+            |}
+            |
+            |input [type = 'number']: : -webkit-inner-spin-button , input [type = 'number']: : -webkit-outer-spin-button{
+            |  height : auto ;
+            |}
+            |
+            |input [type = 'search']{
+            |  -webkit-box-sizing : content-box ;
+            |  -moz-box-sizing : content-box ;
+            |  box-sizing : content-box ;
+            |  -webkit-appearance : textfield ;
+            |}
+            |
+            |input [type = 'search']: : -webkit-search-cancel-button , input [type = 'search']: : -webkit-search-decoration{
+            |  -webkit-appearance : none ;
+            |}
+            |
+            |@font-face{
+            |  font-family : 'Glyphicons Halflings' ;
+            |  src : url('../fonts/glyphicons-halflings-regular.eot') ;
+            |  src : url('../fonts/glyphicons-halflings-regular.eot?#iefix') format ('embedded-opentype'), url('../fonts/glyphicons-halflings-regular.woff2') format ('woff2'), url('../fonts/glyphicons-halflings-regular.woff') format ('woff'), url('../fonts/glyphicons-halflings-regular.ttf') format ('truetype'), url('../fonts/glyphicons-halflings-regular.svg#glyphicons_halflingsregular') format ('svg');
+            |}
+            |
+            |h4 small , . h4 small , h5 small , . h5 small , h6 small , . h6 small , h4 . small , . h4 . small , h5 . small , . h5 . small , h6 . small , . h6 . small{
+            |  font-size : 75% ;
+            |}
+            |
+            |@media (min-width : 768px){
+            |  . lead {
+            |    font-size : 21px ;
+            |  }
+            |}
+            |
+            |@media screen and (max-width : 767px){
+            |  . table-responsive {
```

Comment:
Pretty sure we need to have no space after the `.` when selecting on a css-class